documentation/faq/700-integrations.md

```diff
@@ -0,0 +1,15 @@
+---
+question: How do I use X with SvelteKit?
+---
+
+## How do I setup library X?
+
+Please see [sveltejs/integrations](https://github.com/sveltejs/integrations#sveltekit) for examples of using many popular libraries like Tailwind, PostCSS, Firebase, GraphQL, mdsvex, and more.
```

Comment:
We could also refer to the `svelte-adders` repo here

Reply:
It's actually within the integrations repo link (and is what the list of popular libraries based on).

Reply:
Ah ok, nevermind then